csfieldguide/general/context_processors/nav_context.py

```diff
@@ -9,4 +9,12 @@ def nav_context(context):
     Returns:
         Dictionary containing list of chapter objects
     """
-    return {'chapters': Chapter.objects.all()}
+    chapters = Chapter.objects.all()
+    # Add first chapter section to each chapter
+    chapter_sections = ChapterSection.objects.all()
+    for chapter in chapters:
```

Comment:
This seems really inefficient, as it's getting the first item for every chapter, just to have one chapter's title. Unless I'm missing the point of this completely?